codebuild/bin/s2n_codebuild.sh

```diff
@@ -85,8 +85,12 @@ setup_apache_server() {
 run_integration_v2_tests() {
     setup_apache_server
     "$CB_BIN_DIR/install_s2n_head.sh" "$(mktemp -d)"
-    make clean
-    make integrationv2
+    cmake -DPython3_EXECUTABLE=$(which python3) .
```

Comment:
when running this locally, I'm getting the OS libcrypto.  take a look at the unit test function starting on line 92 for the arguments used there, guessing many/most of those will be required here.